subjects/replaceeven/README.md

```diff
@@ -0,0 +1,21 @@
+## replaceeven
+
+### Instructions
+
+Write a program that takes every character with an even index, of each argument and replace it with the number 2.
+
+- you program should always print a newline `\n` at the end.
+- you program should print all the arguments in one line, separated by the space character `' '`.
+- `0` is an even number.
+
+
+### Usage
+
+```console
+$ go run . | cat -e
+$
+$ go run . hello 444444444444444 | cat -e
+2e2l2 242424242424242$
+$ go run main.go "Zone 01 is the best" | cat -e
```

Comment:
Instead of this -->  go run main.go "Zone 01 is the best" | cat -e
This should be --> go run . "Zone 01 is the best" | cat -e

Reply:
nice catch. Fixed